README.md

```diff
@@ -108,23 +108,21 @@ join require identification. To use this method enter your nick password to the
 
 ### Using external commands for passwords
 
-When a password field in the config file starts with `$`, tiny uses rest of the
-field as the shell command to run to get the password.
+When a password field in the config file is a map with a `cmd` key, the value
```

Comment:
Should we update the sample `config.yml` too?

Reply:
We show an example in the README so I'm not sure if it's necessary.